solutions/devsprint-bruno-faganello-2/SpaceApp/Screens/DetailsView/DetailsView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DetailsView.swift
+//  SpaceApp
+//
+//  Created by Bruno Silva on 27/04/22.
+//
+
+import SwiftUI
+
+struct DetailsView: View {
+    
```

Comment:
[REQUEST CHANGED]
Creating a parameter to set the details information and a action to return button clicked